commit 4ccebbfa2f27d08fbdf55c1017ef70cb42a0e2f9
Author: Adam C. Emerson <aemerson@redhat.com>
Date:   Sat Mar 7 04:33:10 2020 -0500

    mgr: Build target 'common' without using namespace in headers
    
    Part of a changeset to allow building all of 'common' without relying
    on 'using namespace std' or 'using namespace ceph' at toplevel in
    headers.
    
    Signed-off-by: Adam C. Emerson <aemerson@redhat.com>

diff --git a/src/mgr/MgrClient.cc b/src/mgr/MgrClient.cc
index 4903fa3a083..2f998635c3f 100644
--- a/src/mgr/MgrClient.cc
+++ b/src/mgr/MgrClient.cc
@@ -33,6 +33,9 @@ using std::string;
 using std::vector;
 
 using ceph::bufferlist;
+using ceph::make_message;
+using ceph::ref_cast;
+using ceph::ref_t;
 
 #define dout_subsys ceph_subsys_mgrc
 #undef dout_prefix
diff --git a/src/mgr/MgrClient.h b/src/mgr/MgrClient.h
index 6ac56fec970..aed1072c674 100644
--- a/src/mgr/MgrClient.h
+++ b/src/mgr/MgrClient.h
@@ -126,7 +126,7 @@ public:
   bool handle_mgr_close(ceph::ref_t<MMgrClose> m);
   bool handle_command_reply(
     uint64_t tid,
-    bufferlist& data,
+    ceph::buffer::list& data,
     const std::string& rs,
     int r);
 
@@ -151,7 +151,7 @@ public:
     ceph::buffer::list *outbl, std::string *outs,
     Context *onfinish);
   int start_tell_command(
-    const string& name,
+    const std::string& name,
     const std::vector<std::string>& cmd, const ceph::buffer::list& inbl,
     ceph::buffer::list *outbl, std::string *outs,
     Context *onfinish);
